### DISCUSSION TITLE
Is there any sample to run couchdb on k8s with prometheus operator

### DISCUSSION THREAD

**Discussion by SamYuan1990** [apache/couchdb#3935] [Q&A]
as subject.

- [ ] https://docs.couchdb.org/en/stable/config/misc.html#config-prometheus how to ref this part config/enable prometheus endpoint for couchdb port?
- [ ] https://docs.couchdb.org/en/stable/api/server/common.html#api-server-prometheus then we can use the prometheus endpoint integrated with prometheus operator to make prometheus data been send/pulled between couchdb and prometheus?

**Comment by kocolosk** [comment]
It's not super well-documented but the CouchDB Helm chart does allow you to enable the prometheus endpoint. The Pull Request that introduced the support is probably the best documentation:

https://github.com/apache/couchdb-helm/pull/57

**Reply by SamYuan1990** [reply]
hi @kocolosk , I am not sure if there any possible way to set up ADDITIONAL_PORT as COUCHDB_USER by environment values? it will more easy to use environment instead of configuration file in docker and k8s.
